src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        if (fromFileName== null || toFileName == null){
+            return;
+        }
+
+        String sCurrentLine;
+        Integer supplyCount = 0;
+        Integer buyCount = 0;
+
+        try (BufferedReader br = new BufferedReader(new FileReader (fromFileName))) {
+
+            while ((sCurrentLine = br.readLine()) != null)
+            {
+                String [] elements = sCurrentLine.split( ",");
+                if (elements[0].equals("supply")){
+                    supplyCount += Integer.parseInt(elements[1]);
+                } else {
+                    buyCount += Integer.parseInt(elements[1]);
+                }
```

Comment:
There is no validation for the format of the file content. If the file contains unexpected data, `Integer.parseInt` could throw a NumberFormatException. Consider adding validation to ensure the data is in the expected format before parsing.